Dockerfile

```diff
@@ -1,19 +1,25 @@
 # vim:set ft=dockerfile:
 FROM condaforge/mambaforge
-MAINTAINER https://github.com/bird-house/finch
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PIP_ROOT_USER_ACTION=ignore
+LABEL org.opencontainers.image.authors="bourgault.pascal@ouranos.ca"
 LABEL Description="Finch WPS" Vendor="Birdhouse" Version="0.11.1"
 
+# Switch to /code directory
 WORKDIR /code
+COPY . ./
 
+# Build finch environment
 COPY environment.yml .
-RUN mamba env create -n finch -f environment.yml \
-    && mamba install -n finch gunicorn psycopg2 \
-    && mamba clean -ay
```

Comment:
Keep all install steps in the same `RUN` layer for proper clean up at the end.  If you perform in different layers, then the final clean up is useless since it can not clean up in other layers that itself.